pkgs/development/compilers/rust/1_39_0.nix

```diff
@@ -10,6 +10,7 @@ import ./default.nix {
   bootstrapHashes = {
     i686-unknown-linux-gnu = "41aed8a350e24a0cac1444ed99b3dd24a90bc581dd88cb420c6e547d6b5f57af";
     x86_64-unknown-linux-gnu = "adda26b3f0609dbfbdc2019da4a20101879b9db2134fae322a4e863a069ec221";
+    arm-unknown-linux-gnueabihf = "0603a3d3d16ae8f3b3b117eb699e8f3ef7532a6f6d3c29d13e7d4614fc3c9e7a";
```

Comment:
Can you also update `print-hashes.sh` to include this?

Reply:
I already did, see: https://github.com/NixOS/nixpkgs/pull/73472/files#diff-94873d51c01d719eb9630e042ad220f0R13